packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -124,7 +124,7 @@ export default class Meetings extends WebexPlugin {
        * @memberof Meetings
        */
 
-      this.meetingInfo = null;
+      this.meetingInfo = this.config.experimental.enableUnifiedMeetings ? new MeetingInfoV2(this.webex) : new MeetingInfo(this.webex);
```

Comment:
I've tested this and `this.config` is actually not available at the point when constructor is called, so this fix doesn't work. However, I think I've found the right place to do this. When I've moved this line of code into the callback for `READY` event that's registered in the `onReady()` method, then it works for me.

Reply:
i was worried about that... was gonna see if the tests caught it and then investigate it

Reply:
thanks for diggin in @marcin-bazyl !